common/script_values/00_men_at_arms_values.txt

```diff
@@ -365,3 +315,177 @@ culture_ai_weight_heavy_cavalry = {
 }
 
 # Warcraft
+
+culture_ai_weight_caster = {
+	value = 0
+	
+	if = { # Cultures that favours casters
+		limit = {
+			OR = {
+				culture_group = culture_group:human_group
+				culture_group = culture_group:dwarf_group
+				culture_group = culture_group:orc_group
+			}
+		}
+		add = 40
+	}
+	if = { # Cultures that dislike casters
+		limit = {
+			OR = {
+
+			}
+		}
+		add = -60
+	}
+	
+	if = { # Cultures with a special MaA of that type should avoid the base version of the unit
+		limit = {
+			culture = {
+				OR = {
+
+				}
+			}
+		}
+		add = -100
+	}
+}
+culture_ai_weight_massive_infantry = {
+	value = 0
+	
+	if = { # Cultures that favours giants
+		limit = {
+			OR = {
+				culture_group = culture_group:dwarf_group
+				culture_group = culture_group:undead_group
+				culture = culture:goblin
+			}
+		}
+		add = 40
+	}
+	if = { # Cultures that dislike giants
+		limit = {
+			OR = {
+				culture_group = culture_group:human_group
+				culture_group = culture_group:gnome_group
+			}
+		}
+		add = -60
+	}
+	
+	if = { # Cultures with a special MaA of that type should avoid the base version of the unit
+		limit = {
+			culture = {
+				OR = {
+
+				}
+			}
+		}
+		add = -100
+	}
+}
+culture_ai_weight_light_aerial = {
+	value = 0
+	
+	if = { # Cultures that favours light aerial
+		limit = {
+			OR = {
+				culture_group = culture_group:human_group
+				culture_group = culture_group:elf_group
+				culture_group = culture_group:dwarf_group
+				culture_group = culture_group:harpy_group
+				culture_group = culture_group:dragon_group
+			}
```

Comment:
Harpies should really like aerial together with dragons.
Dragonmaw orcs and Wildhammer should favour aerial. Other cultures are doubtful, aerial is never an important part of the human army.